ipam/allocator.go

```diff
@@ -413,6 +418,10 @@ func (a *Allocator) getPredefinedPool(as string, ipV6 bool) (*net.IPNet, error)
 		}
 	}
 
+	if !supported {
```

Comment:
No, because `getPredefined()` returns the list of predefined pools, not the list of available predefined pools.